.gitallowed

```diff
@@ -0,0 +1 @@
+public static final String TESTING_PRIVATE_KEY = .*
```

Comment:
This looks interesting. What does this do?

Reply:
The presence of a PEM-encoded private key in one of the tests is triggering `git-secrets`. This instructs `git-secrets` to ignore the pattern.